samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,394 @@
+Additionally, this sample is an early version and is only meant to showcase the start of Nordic Semiconductor’s implementation of the SUIT DFU procedure.
+
+User interface
+**************
+
+LED 1:
+    The number of blinks indicates the application SUIT envelope sequence number.
+    The default is set to ``1`` to blink once, indicating "Version 1".
+
+Configuration
+*************
+
+|config|
+
+The default configuration uses UART with sequence number 1 (shown as Version 1 in the nRF Device Manager app).
+To change the sequence number of the application, configure the :ref:`CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM` Kconfig option.
+This also changes the number of blinks on **LED 1** and sets the :ref:`sequence number <ug_suit_dfu_suit_manifest_elements>` of the SUIT :ref:`envelope <ug_suit_dfu_suit_concepts>`’s manifest.
+
+To use this configuration, include the ``SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=x`` option with the build command, where x is the version number.
+
+For example:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
+
+If you do not specify this configuration, the sample is built with sequence number 1 (shown as Version 1 in the nRF Device Manager app.
+
+Configuration options
+=====================
+
+Check and configure the following configuration option for the sample:
+
+.. _CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM:
+
+CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM - Configuration for the sequence number.
+   The sample configuration updates the sequence number of the SUIT envelope, which is reflected as the version of the application in the nRF Device Manager app.
+   The default value is ``1``.
+
+Modify partition sizes
+======================
+
+You can also modify the size and location of the partitions.
+This is done by modifying the values for the desired location and size of the partition in the devicetree :file:`.overlay` files.
+
+* To modify the application core’s partition size,  modify the values for ``cpuapp_slot0_partition`` defined in the :file:`zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi` file within the Zephyr repository.
+
+* To modify the DFU partition, modify the values for ``dfu_partition`` defined in :file:`samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp.overlay`.
+
+    * ``dfu_partition`` is where the update candidate is stored before the update process begins.
+
+Manifest files
+==============
+
+The SUIT DFU procedure requires an envelope to transport the firmware update, and SUIT envelopes require a SUIT manifest.
+All required manifest files (used to later create SUIT envelopes) are automatically created after the sample is built for the first time, and are the following:
+
+* The root manifest - :file:`root_hierarchical_envelope.yaml.jinja2`
+
+* The application domain manifest - :file:`app_envelope.yaml.jinja2`
+
+* The radio domain manifest - :file:`rad_envelope.yaml.jinja2`
```

Comment:
We can now link to output build files here too! :)

Reply:
Ah, yeah! Will add.